serving-sdk/mvnw.cmd

```diff
@@ -0,0 +1,187 @@
+@REM ----------------------------------------------------------------------------

```

Comment:
I'm not a Java programmer but could you please describe briefly what the new files are, how they are used and how they were generated?

Reply:
They are generated by running `mvn wrapper:wrapper -Dmaven=3.8.6` inside `serving-sdk` directory. It generates `mvnw` and `mvnw.cmd` scripts that are maven wrappers and replace `mvn` command. `mvnw` is used on unix systems and `mvnw.cmd` is used on windows. These wrappers invoke `maven-wrapper.jar` which downloads maven distribution  specified in `maven-wrapper.properties` if not already in cache, installs it, then launches maven. Updating maven version is as simple as updating version in `maven-wrapper.properties` and works on all platforms.